core/http/src/test/java/org/trellisldp/http/WebAcFilterTest.java

```diff
@@ -186,4 +186,9 @@ public void testFilterResponseBaseUrl() throws Exception {
         assertEquals("acl", link.getRel());
         assertEquals("http://example.com/?ext=acl", link.getUri().toString());
     }
+
+    @Test
+    public void testNoopFilterCtor() {
+        assertDoesNotThrow(() -> new WebAcFilter());
```

Comment:
Or `         assertDoesNotThrow(WebAcFilter::new);`? You know I have a serious fetish for point-free style. :grin:

Reply:
Actually, that's what I first wrote, and the compiler didn't like it, so I had to use the longer form.

Reply:
Very interesting (although obviously not important). I might look into that after you merge this. I wonder why?

Reply:
My suspicion is that the `assertDoesNotThrow` [method](https://junit.org/junit5/docs/5.3.0/api/org/junit/jupiter/api/Assertions.html) accepts two types of arguments, both of which are functional interfaces, and it can't decide between an `Executor` and a `ThrowingSupplier<T>`.